en/08_Changelogs/5.3.0.md

```diff
@@ -9,7 +9,7 @@ title: 5.3.0 (unreleased)
 - [Features and enhancements](#features-and-enhancements)
   - [High-level API for converting files](#file-converter)
   - [Improve customisability of rendered images](#image-rendering)
-  - [Validation for elemental content blocks when saving individual blocks](#elemental-validation)
+  - [Validation for inline-editable elemental content blocks](#elemental-validation)
```

Comment:
```suggestion
  - [Validation for inline-editable elemental blocks](#elemental-validation)
```
"Content block" is a specific class, so we should just say "elemental blocks". The actual heading will need to update, as well.

Reply:
Updated

Reply:
Still need to update the actual heading

Reply:
@emteknetnz Still not done?

Reply:
Updated